packages/mobile/src/components/core/FixedFooter.tsx

```diff
@@ -0,0 +1,89 @@
+import type { ReactNode } from 'react'
```

Comment:
I know we have quite a few implementations of this, could you spend a sec reviewing all the implementations and determining if they could be reused here or whether this should be the standard going forward?